### DISCUSSION TITLE
Trino stops running just after 20s from starting it

### DISCUSSION THREAD

**Discussion by sanatriki** [trinodb/trino#18186] [Q&A]
i use trino version 419 and after starting trino using /launcher start and checking the UI http://localhost:8080/ui/,  it works successfully then after 20s trino shuts down and stops and i get this error in the UI : 

Unable to connect to server
Error: TypeError: NetworkError when attempting to fetch resource.

what is the problem please ? thanks!

**Comment by jiangzhx** [comment]
Could you please share some logs from the Trino coordinator server?
